src/tenant/components/contact/Contact.css

```diff
@@ -1,13 +1,11 @@
 
   
-  form{
+  /*form{
```

Comment:
this classname will likely affect all other forms, change name to sth like "contact-form"